app/src/main/java/org/simple/clinic/medicalhistory/SelectDiagnosisErrorDialog.kt

```diff
@@ -0,0 +1,31 @@
+package org.simple.clinic.medicalhistory
+
+import android.app.Dialog
+import android.os.Bundle
+import androidx.appcompat.app.AlertDialog
+import androidx.appcompat.app.AppCompatDialogFragment
+import androidx.fragment.app.FragmentManager
+import org.simple.clinic.R
+
+class SelectDiagnosisErrorDialog : AppCompatDialogFragment() {
+
+  companion object {
+
+    private const val FRAGMENT_TAG = "select_diagnosis_error_alert"
+    fun show(fragmentManager: FragmentManager) {
+      val fragment = SelectDiagnosisErrorDialog().apply {
+        isCancelable = false
```

Comment:
Is there a need to make this uncancelable? I think it's fine since tapping on the "Save" button with bring up the dialog again.

Reply:
Fair point